libs/import/src/lib/view-extracted-elements/contract/utils.ts

```diff
@@ -177,47 +199,51 @@ export async function formatContract(
         };
         return centralOrgId.catalog;
       } else {
+        if (!value) return '';
         const isInternal = data.contract.sellerId === centralOrgId.catalog;
-        const sellerId = await getOrgId(value, orgService, orgNameCache);
-        if (isInternal && !sellerId) return unknownEntityError('Licensee Organization');
-        return sellerId;
+        let buyerId = await getOrgId(value, orgService, orgNameCache);
+        if (!buyerId && await verifyIdExists(value, titleService)) buyerId = value;
+        if (isInternal && !buyerId) return unknownEntityError('Licensee Organization');
+        return buyerId;
```

Comment:
return as soon as you can : 
```suggestion
        const buyerId = await getOrgId(value, orgService, orgNameCache);
        if (buyerId) return buyerId;
        if (await verifyIdExists(value, titleService)) return value;
        if (isInternal) return unknownEntityError('Licensee Organization');
        return ''; // maybe return an error if it's mandatory ?
```